faststream/kafka/producer.py

```diff
@@ -35,16 +35,12 @@ async def publish(
 
         headers_to_send = {
             "content-type": content_type or "",
+            "correlation_id": correlation_id or str(uuid4()),
```

Comment:
correlation_id is still in the signature of the publish(), but it is not used anywhere.

Reply:
I was just wondering if that was a bug. The parameter is specified in the signature of the function and it is not used anywhere in the body.